checks.d/mysql.py

```diff
@@ -97,8 +97,8 @@ def _get_config(self, instance):
 
     def _connect(self, host, port, mysql_sock, user, password, defaults_file):
         service_check_tags = [
-            'host:%s' % host,
-            'port:%s' % port
+            'host:%s' % mysql_sock if mysql_sock != '' else host,
```

Comment:
that makes sense, I'm tempted to go with it but at the same time don't want to change the behavior too much if a lot of people are relying on this value. Thoughts @remh ?